docs/requirements.txt

```
@@ -6,4 +6,4 @@ setuptools==73.0.1
 
 # These requirements must match with the Baseline API version documented in the Installation Guide.
 numpy==1.26.*
-scipy==1.11.*
+scipy==1.14.*
```

Should not also this one?

---

Thanks for the catch! Absolutely, this shouldn't be in.
```suggestion
scipy==1.11.*
```